test/controllers/old_node_controller_test.rb

```diff
@@ -188,29 +188,31 @@ def test_current_version
   # authorised.
   def test_redact_node_unauthorised
     do_redact_node(nodes(:node_with_versions_v3),
-                   redactions(:example))
+                   create(:redaction))
     assert_response :unauthorized, "should need to be authenticated to redact."
   end
 
   ##
   # test the redaction of an old version of a node, while being
   # authorised as a normal user.
   def test_redact_node_normal_user
-    basic_authorization(users(:public_user).email, "test")
+    user = create(:user)
+    basic_authorization(user.email, "test")
 
     do_redact_node(nodes(:node_with_versions_v3),
-                   redactions(:example))
+                   create(:redaction, :user => user))
```

Comment:
For the tests that succeed, there was previously a correlation in that the fixture redaction was owned by `users(:moderator_user)`, which was the same user as the one used for logging in. So I thought it best to make those ones owned by the same moderator, and therefore even the failing ones like this should be owned by the same user, just in case.

But you're right, of course, this isn't actually required. I didn't check the controller code.

There's also a design issue here that I had in the back of my mind. For redactions it is currently possible for moderators to use a redaction created by someone else, and even edit it. But for user blocks, only the moderator that created one can edit it. So the "ownership" behaviour of user_blocks is quite different from redactions. There might be advantages in aligning the behaviour one way or the other in future.

If you'd prefer that I remove these (currently?) unnecessary ownerships in the tests, I'm happy to do that.